components/templates/src/builtins/summary-cutoff.html

```diff
@@ -0,0 +1 @@
+{% if summary is matching("\PP$") %}&hellip;{% endif %}
```

Comment:
Can we default to an ellipsis without the test and mention that in the docs as an example?
`matching` will currently create a new regex everytime this template is called so it's not ideal perf wise as a default

Reply:
Oh, I had no idea that was the case; I'm fine with that, then.

Reply:
FWIW, no idea where Tera 2 is currently, but I'd be willing to help figure out how to solve this problem there. Feels like it should be doable to let the functions know about the constant arguments and optimize themselves accordingly.

Reply:
It's possible to fix in tera 1 I think yes, just have a cache for the compiled regexes